packages/crypto/src/transactions/types/schemas.ts

```diff
@@ -215,9 +215,23 @@ export const multiPayment = extend(transactionBaseSchema, {
 
 export const delegateResignation = extend(transactionBaseSchema, {
     $id: "delegateResignation",
+    required: ["asset"],
     properties: {
         type: { transactionType: TransactionTypes.DelegateResignation },
         amount: { bignumber: { minimum: 0, maximum: 0 } },
+        asset: {
```

Comment:
Why do you need an asset? The transaction will be applied to the sender which is required to be a wallet with a username so you can grab the username from it directly.